### ISSUE TITLE
Val loss mismatch with train loss

### ISSUE THREAD

**Issue by sachit-menon** [AntixK/PyTorch-VAE#7]
Hi, thanks for this repository. I noticed while running that the train loss numbers are around <0.1 after an epoch, but the validation loss ranges from 20-30. I think this mismatch was introduced by #2 - the division for `M_N` now uses a different denominator in train vs val. The original intent was to get around `val` being run for the first time before `train` in the newer versions of lightning, but I don't think this is correct. Instead, one workaround is to run `Trainer.fit` with `num_sanity_val_steps=0`; this way `num_train_steps` is set before validation is run, so the train loss and val loss are back on the same scale. By doing this, I get similar numbers for both. 

Am I misinterpreting something? Please let me know if I'm incorrect/the train and val losses should be very different. I'm not sure I understand the different scaling.
